### ISSUE TITLE
Train Model 3: Importance Adjuster (2M params)

### ISSUE THREAD

**Issue by marctjones** [marctjones/klareco#667]
## Goal

Train a small learned model to fine-tune deterministic importance scores, handling cases where schema/RST/novelty formula is insufficient.

## Architecture

```python
class ImportanceAdjuster(nn.Module):
    # 2M parameters
    # Inputs: fact embeddings, query embedding, context, deterministic_score
    # Output: adjusted importance score (deterministic ± 0.2)
```

## When It Helps

1. **Query relevance**: Semantic match not captured by schema
   - Example: "kuracisto" answers "kiu" (WHO) questions
2. **Redundancy**: Paraphrase detection across facts
   - Example: "kreis" vs "fondis" Esperanto (same meaning)
3. **Frequency-based surprise**: Rare words = high information content
   - Example: "inventis" (tier2) stronger signal than "faris" (tier0)

## Training Data Needed

5,000-10,000 examples: (fact, context, query, gold_importance)

**Collection strategy** (active learning):
1. Run deterministic pipeline on 100 queries
2. Human evaluation: identify where deterministic scores are wrong
3. Annotate correct importance for those cases
4. Train model to predict corrections

## Training Approach

```python
# Loss: MSE between predicted and gold adjustment
gold_adjustment = human_importance - deterministic_importance
predicted_adjustment = model(fact, query, context, deterministic_score)
loss = nn.MSELoss()(predicted_adjustment, gold_adjustment)
```

## Quality Metrics

- Correlation with human judgments: >0.80
- Mean adjustment magnitude: <0.15 (mostly small corrections)
- Improved summary quality: +5-10% over pure deterministic

## Dependencies

- #666 (Phase 0 validation - measure where deterministic fails)
- Root Embeddings (#479, should be retrained first)
- Deterministic baseline implemented

## Timeline

2 weeks (Phase 2, Week 3-4)

## Optional

Can be disabled to compare deterministic vs learned performance.

## Related

- See: docs/COMPLETE_SYSTEM_DESIGN_WITH_MODELS.md (Model 3)
- Part of Epic #654

**Comment by marctjones** [comment]
Closing as part of a 2026-09-08 backlog cleanup (see PLAN.md, DESIGN.md, CLAUDE.md's merge gate).

This issue predates the merge gate (adopted 2026-07-13) and the AST-native orchestrator rewrite. It describes work against the pre-orchestrator RAG pipeline or the learned model stack (M1/M2 embeddings, compositional embeddings, entity classifiers, neural rerankers) that was pruned from the repo -- klareco/embeddings/, klareco/models/, and klareco/summarization/ no longer exist. It has no milestone and was never benchmarked, matching exactly the failure mode CLAUDE.md documents (25 capability issues filed 2026-05-26, 8 implemented, none benchmarked).

Labeled archived-2026-09 so this batch is recoverable: gh issue list --state closed --label archived-2026-09. If this capability is still wanted, re-file it against the current architecture naming the benchmark metric and test the merge gate requires.
